src/main/resources/sql/monitoringLocation/readBiodataMonitoringLocation.sql

```diff
@@ -25,7 +25,7 @@ select
 	biodata_site.alt_datum_cd,
 	station_nwis.elevation_method,
 	station_nwis.vdatum_id_code,
-	coalesce(station_nwis.drain_area_value, to_number(biodata_site.drain_area_va, '12345678')) drain_area_va,
+	station_nwis.drain_area_value drain_area_va,
```

Comment:
This is a change in logic - tho old code used the value from biodata if the nwis value  is null

Reply:
Yup, rather than do the coalesce and number conversion here, I do it in the processor.  Happy to revisit the number conversion logic here if desired/needed though.